src/Pools/Pool.php

```diff
@@ -32,19 +32,19 @@ class Pool
     protected int $reconnectSleep = 1; // seconds
 
     /**
-     * @var array
+     * @var array<int, true>|array<Connection>
```

Comment:
When does it have two different structures for the array ?

Also does the IDE even benefit from `array<int, true>` ?? Is the IDE able to use the additional information `<int, true>`

Reply:
Its there so PHP Stan (cod eanalyser` does not complain.

Notice in constructor we fill it with `true` values. Thats the first part.
But then when you try to `pop`, it is getting set to Connection. Thats the second part.

Reply:
Changed to `@var array<Connection|true>`, that explains this scenario better.

Reply:
Alright 👍